### PR TITLE
[vulkan*] update to 1.3.275.0

### PR DESCRIPTION
**Pull request by moritz-h**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/lunarg-vulkantools/vcpkg.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "lunarg-vulkantools",
-  "version": "1.3.268.0",
+  "version": "1.3.275.0",
   "description": "Vulkan header files and API registry",
   "homepage": "https://github.com/KhronosGroup/Vulkan-Headers",
```
Comment (dg0yt) [inline ports/lunarg-vulkantools/vcpkg.json:5]: Do we need to fix description and homepage here?

**Comment by moritz-h** [comment]
Is there a way to get the error logs from the CI to see what happens on x64_android? At least for everything else the CI is green now.
What I'm not fully sure about is setting `lunarg-vulkantools` to not support staticcrt. Inside vulkan layers are build which are a MODULE library. Is there a better way to solve this?
Also there are a few hacks for android in `vulkan-tools`, because `vkcube` target is a tool on all other platforms, but a library on android.

**Comment by MonicaLiu0311** [comment]
Please get [failure logs here](https://dev.azure.com/vcpkg/c1ee48cb-0df2-4ab3-8384-b1df5a79fe53/_apis/build/builds/98758/artifacts?artifactName=failure%20logs%20for%20x64-android&api-version=7.1&%24format=zip):
```
-- The C compiler identification is Clang 14.0.7
-- The CXX compiler identification is Clang 14.0.7
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /android-ndk-r25c/toolchains/llvm/prebuilt/linux-x86_64/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /android-ndk-r25c/toolchains/llvm/prebuilt/linux-x86_64/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Git: /bin/git (found version "2.25.1") 
CMake Error at windows-runtime-installer/CMakeLists.txt:22 (configure_file):
  No such file or directory


-- Configuring incomplete, errors occurred!
```

You can also get specific error logs by following the steps below:
image
image
image
image

**Comment by dg0yt** [comment]
> ~~~
> configure_file(
>     ${CMAKE_CURRENT_LIST_DIR}/VulkanRT-License.txt.in
>     ${CMAKE_CURRENT_LIST_DIR}/VulkanRT-License.txt
>     @ONLY
> )

This writes to the source dir. This results in a race condition in vcpkg when release and debug are configured in parallel. Patch, or pass `DISABLE_PARALLEL_CONFIGURE` to `vcpkg_cmake_configure`.

**Comment by moritz-h** [comment]
Ok thanks. I decided for a patch, as this directory only updates the version year in the installer generation files, but generating an installer is unused here. So removing parallel configure to increase build times is probably more annoying.

**Pull request merged**